### ISSUE TITLE
bug: A11y testing workflow fails randomly

### ISSUE THREAD

**Issue by ethanWallace** [cds-snc/gcds-docs#528]
The new accessibility testing workflow for the documentation site can randomly fail on two pages causing the author of a pull request to have to rerun the tests. The tests randomly fail on the basic page template preview pages.

## Example

```
3 accessibility violations were detected with moderate and serious impacts.
┌─────────┬───────────────────────────────┬────────────┬────────────────────────────────────────────────────────────────────────────────────┬───────┐
│ (index) │ id                            │ impact     │ description                                                                        │ nodes │
├─────────┼───────────────────────────────┼────────────┼────────────────────────────────────────────────────────────────────────────────────┼───────┤
│ 0       │ 'landmark-one-main'           │ 'moderate' │ 'Ensure the document has a main landmark'                                          │ 1     │
│ 1       │ 'page-has-heading-one'        │ 'moderate' │ 'Ensure that the page, or at least one of its frames contains a level-one heading' │ 1     │
│ 2       │ 'scrollable-region-focusable' │ 'serious'  │ 'Ensure elements that have scrollable content are accessible by keyboard'          │ 1     │
└─────────┴───────────────────────────────┴────────────┴────────────────────────────────────────────────────────────────────────────────────┴───────┘
    1) page Templates Basic Preview: /fr/modeles-de-page/basic/apercu```

**Comment by adorayi** [comment]
Investigation required.
